src/models/bookmark.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+    const BookMark = sequelize.define('BookMark', {
+        articleId: DataTypes.INTEGER,
+        userId: DataTypes.INTEGER,
+        name: DataTypes.STRING,
+        collection: DataTypes.STRING
+    }, {});
+    BookMark.associate = function (models) {
+        BookMark.belongsTo(models.Article, { foreignKey: 'articleId' })
```

Comment:
@Anguandia what do you think when BookMark and Article relationship has onUpdate: 'CASCADE', and
	      onDelete: 'CASCADE?